### ISSUE TITLE
grid view websocket events for underlying view are not handled

### ISSUE THREAD

**Issue by teosarca** [metasfresh/metasfresh-webui-frontend#387]
Test case:
* open receipt candidates
* select a line and receive some HU(s)
* create material receipt from one of them

At this moment, the backend is firing an websocket event like following:
topic: /view/a429a03c-9d74-4e92-9c03-609a5625fc78
event: {viewId=a429a03c-9d74-4e92-9c03-609a5625fc78, windowId=540196, fullyChanged=true, changedIds=[1000001]}

Because the underlying receipt candidate is was changed (i.e. QtyMoved increased, QtyToMove decreased).
But it seems that the frontend is no longer listening to that topic.

![image](https://cloud.githubusercontent.com/assets/1244701/23165234/c1dd88ea-f843-11e6-8d65-264e1d598d2a.png)

**Comment by teosarca** [comment]
note: my colleagues told me that sometimes works sometimes not.
in meantime i am checking the backend side too.

**Comment by damianprzygodzki** [comment]
I've played with that a bit. It is not reproducable. 

Always when the event is coming and has `fullyChanged` property set the app is getting documentView with its ID.

Furthermore, the documentView is always listening WS until it is unmounted (it means that component is destroyed).

Checked by quickActions and actions in nav menu.

![screen shot 2017-02-23 at 10 20 42](https://cloud.githubusercontent.com/assets/5701744/23252684/0dab7a74-f9b2-11e6-8b50-9c5fd8a3a775.png)
